ci/release.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
How about kotlin script? 
It'd be easier to maintain.

Reply:
i don't mind to try kts here